.circleci/config.yml

```diff
@@ -134,12 +136,56 @@ jobs:
       - run:
           name: Building Plugin Bundle
           command: make dist
+      - run:
+          name: Generating Release Notes
+          command: |
+            printf "Supported Mattermost Server Versions: **$(cat plugin.json | jq .min_server_version -r)+** \n## Enhancements\n\n## Fixes\n"  >> dist/release-notes.md
```

Comment:
Are the "Enhancements" ever getting written this way? I am totally fine with just having the raw list of commits at this point.

Reply:
Good point! Removed that, and just renamed it to `Commits`.